src/Components/redux/configuration.js

```diff
@@ -0,0 +1,17 @@
+import { combineReducers, configureStore } from '@reduxjs/toolkit';
+import booksReducer from './books/books';
+import categoriesReducer from './categories/categories';
+
+// combine all reducers together
+
+const reducer = combineReducers({
+  book: booksReducer,
```

Comment:
- [ ] [OPTIONAL] Line 8 `book` should be plural as in "books" instead.

Reply:
Thanks for the suggestion, file has been updated. Thank you.